ui/js/ui.transfers.js

```diff
@@ -81,7 +81,18 @@ var UI = (function(UI, $, undefined) {
           return;
         }
 
-        iota.api.sendTransfer(connection.seed, connection.depth, connection.minWeightMagnitude, [{"address": address, "value": amount, "message": "", "tag": tag}], {"inputs": inputs.inputs}, function(error, transfers) {
+        var transfers = [{"address": address, "value": amount, "message": "", "tag": tag}];
+        var outputsToCheck = transfers.map(transfer => { return {address: iota.utils.noChecksum(transfer.address)}});
+        var exptectedOutputsLength = outputsToCheck.length;
+        filterSpentAddresses(outputsToCheck).then(filtered => {
+          if (filtered.length !== exptectedOutputsLength) {
+            UI.isDoingPOW = false;
+            UI.formError("transfer", "sent_to_key_reuse_error", {"initial": "send_it_now"});
+            modal = $("#sent-to-key-reuse-modal").remodal({hashTracking: false, closeOnOutsideClick: false, closeOnEscape: false});
+            modal.open();
+            return;
+          }
+          iota.api.sendTransfer(connection.seed, connection.depth, connection.minWeightMagnitude, transfers, {"inputs": inputs.inputs}, function(error, transfers) {
```

Comment:
inputs is not defined? or is that a global?

Reply:
https://github.com/alon-e/wallet/blob/5c8c6e336336abbe959f521caa346ce6f258891b/ui/js/ui.transfers.js#L65
It's private in callback

Reply:
it's the callback param from `getUnspentInputs`:L65